### PR TITLE
Document how to send public emails

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Fixes https://github.com/srobo/infrastructure-team-minutes/issues/18

We send emails very often, but don't have canonical documentation for how to do it properly.

### CODE REVIEW

File: docs/infrastructure/sending-emails.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+- [Team emails](https://github.com/srobo/team-emails)
+- [Volunteer emails](https://github.com/srobo/volunteer-emails/)
+
+Before an email is sent, it should be reviewed.
+
+1. Write your email, in [markdown](https://guides.github.com/features/mastering-markdown/)
+2. Fork the relevant repository, and open a pull request with your changes. The file should be named based on the expected send date, in a directory corresponding to the competition cycle year.
+3. Request the PR be reviewed. Either by using the 'Reviewers' functionality in GitHub, or by sending a message in Slack.
+4. Confirm there aren't any other emails planning to be sent soon
```
Comment (sedders123) [inline docs/infrastructure/sending-emails.md:27]: Should we clarify what to do if there are other emails being sent soon
```suggestion
4. Confirm there aren't any other emails planning to be sent soon. If there are consider merging the emails
```

File: docs/infrastructure/sending-emails.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+#### 4. Send the email
+
+Once all sections have been completed, click "Send" in the top right. A popup will appear confirming the recipients.
+
+### Gmail
+
+Sending an email through GMail is much easier than [Mailchimp](#mailchimp).
+
+Sending Student Robotics related emails **must** be done using your Student Robotics Google account.
```
Comment (sedders123) [inline docs/infrastructure/sending-emails.md:97]: Worth mentioning how to get an account if they don't have one?
```suggestion
Sending Student Robotics related emails **must** be done using your Student Robotics Google account. If you don't have an SR google account ask a committee member to request the creation of one for you?
```

**Review by sedders123 — COMMENTED** [review]
Thanks so much for putting these together Jake. Looks largely as I expected, left a couple of comments inline :)

**Comment by PeterJCLaw** [comment]
Might be good to link to this from https://github.com/srobo/runbook/edit/main/docs/infrastructure/README.md.

Can we ship this? I think this is likely complete enough to be useful, now it's had a review & fixes. We can always add more later.

**Review by sedders123 — APPROVED** [review]

**Pull request merged**